site/_docs/reference.md

```diff
@@ -2852,6 +2855,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b o p | TRANSLATE(expr, fromString, toString)      | Returns *expr* with all occurrences of each character in *fromString* replaced by its corresponding character in *toString*. Characters in *expr* that are not in *fromString* are not replaced
 | b | TRUNC(numeric1 [, numeric2 ])                  | Truncates *numeric1* to optionally *numeric2* (if not specified 0) places right to the decimal point
 | q | TRY_CAST(value AS type)                        | Converts *value* to *type*, returning NULL if conversion fails
+| h s | UNHEX(varchar)                               | Converts a hexadecimal-encoded *varchar* into bytes, returns NULL if input is invalid hexadecimal-encoded *varchar*
```

Comment:
Simplest doc is 'Equivalent to FROM_HEX'.

Reply:
Due to UNHEX's behaviour is not the same as FROM_HEX when input is not a valid hexadecimal string, I'd like to keep the doc here.